ucm2/sof-soundwire/HiFi.conf

```diff
@@ -1,13 +1,5 @@
 # Use case Configuration for sof-soundwire card
 
-SectionVerb {
-
-	EnableSequence [
-		cset "name='PGA1.0 1 Master Playback Volume' 50"
-	]
-
-}
-
 <sof-soundwire/RT700.conf>
```

Comment:
@libinyang Git message still has the wrong section name, otherwise looks good:
Move the kcontrols initial setting into SectionOnce. These values

Reply:
> @libinyang Git message still has the wrong section name, otherwise looks good:
> Move the kcontrols initial setting into SectionOnce. These values

@kv2019i Fixed. Thanks for pointing it out.